ui/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "vscode-azureextensionui",
-    "version": "0.21.1",
+    "version": "0.21.2",
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
         "@types/fs-extra": {
             "version": "4.0.8",
-            "resolved": "https://registry.npmjs.org/@types/fs-extra/-/fs-extra-4.0.8.tgz",
+            "resolved": "http://registry.npmjs.org/@types/fs-extra/-/fs-extra-4.0.8.tgz",
```

Comment:
Can you do all that stuff to clean out your npm cache? So we don't get these http changes?